### DISCUSSION TITLE
Filtering/sorting through forks

### DISCUSSION THREAD

**Discussion by thany** [community/community#12991] [Other Feature Feedback, Questions, & Ideas]
On any reasonably popular repo, click on "XX forks" in the About pane. What you'll get is a list of every fork that was ever made. That's it.

This is useless, because:

1. I have no way to see when a fork was created.
2. Nor when it was last synced.
3. Nor how many stars/watchers it has.
4. Nor how many (additional) issues it has.

Usecase: on an abandoned repo that is reasonably popular, I need to know (to within reason) which fork is the best one to pick. This might be a combination of recentness of that fork, and popularity by stars/watchers.

The only way to get to that kind of information, is to open each fork and make a judgement of how "good" it is. Measures that could be indicative of a "good" fork could easily be presented on the forks page of the original (abandoned) repo in the form of extra columns.

Frankly, the current list of forks is very lazily made. It's just a list of meaningless names (to most) plonked down, just because "we have to display it" or something. Please, Github, make the list of forks actually useful 🙏🏻

**Comment by jxu** [comment]
(mostly) duplicate of #4545

**Reply by thany** [reply]
My feature request is a little mbit more than that. So I wouldn't call this a duplicate at all, not even mostly.

**Reply by Coramax1** [reply]
> (principalmente) duplicado de [# 4545](https://github.com/github/feedback/discussions/4545)

**Comment by endolith** [comment]
This is a bit helpful: https://addons.mozilla.org/en-US/firefox/addon/lovely-forks/

**Reply by lustickd** [reply]
For chrome/ff/safari there's also https://github.com/refined-github/refined-github

It just adds a button that searches https://useful-forks.github.io/ for the repo but that's good enough for me. Shows how many stars and commits ahead/behind each fork is.

**Comment by Coramax1** [comment]
https://github.com/github/feedback/discussions/12991#discussioncomment-2391003

**Comment by jsoref** [comment]
Could someone please move this to the [general-feedback](https://github.com/github/feedback/discussions/categories/general-feedback) category?

the [lists-feedback](https://github.com/github/feedback/discussions/categories/lists-feedback) category is for a specific feature that, roughly speaking, relates to ⭐s.
